lib/helpers/Profile/userProfileHelpers.js

```diff
@@ -3,25 +3,21 @@ if (Meteor.isClient) {
 	Template.UserProfile.helpers({
 		profile: function() {
 			return Meteor.users.find({
-				_id: id
+				_id: id_user
 			}).fetch()[0].profile;
 		},
 		profimg: function() {
 			return Meteor.users.find({
-				_id: id
+				_id: id_user
 			}).fetch()[0].profile.picturelrg;
 		},
-		help: function(){
-			alert("you need viagra");
-			return
-		},
 		defaultCheck: function(){
 			var results = Meteor.users.find()
 			return Boolean(results.count())
 		},
 		listing: function() {
 			return Listing.find({
-				creator_id: id,
+				creator_id: id_user,
```

Comment:
Identifier 'creator_id' is not in camel case.<br>Identifier 'id_user' is not in camel case.<br>'id_user' is not defined.